pages/dkp/kommander/2.3/clusters/custom-domain-certificate/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: layout.pug
+navigationTitle: Custom domain and certificate
+title: Configure a custom domain and certificate
+menuWeight: 30
+excerpt: Configure a custom domain and certificate for your Management or any Managed/Attached cluster
+beta: false
+enterprise: false
+---
+
+DKP supports configuring a custom domain name for accessing the DKP UI and other platform services. Additionally, you can provide a custom certificate for each domain, or one can be issued automatically by Let's Encrypt, or other certificate authorities supporting the ACME protocol.
+
+The configuration path is different depending on the cluster for which you want to customize the domain or certificate.
+
+[Customize a domain or certificate in the **Management Cluster**][management] (during installation).
+
+[Customize a domain or certificate in a **Managed Cluster**][managed] (after installation).
+
+## Why to customize domains
+
+DKP’s default configuration supports the customization of domains to allow you to use your own domain or hostname for your services.
```

Comment:
I wouldn't say that "default configuration supports" something, but rather something like "DKP supports the customization of domain ... ". By default, we don't install DKP with custom domain.

Reply:
```suggestion
DKP supports the customization of domains to allow you to use your own domain or hostname for your services.
```